conjure-lib/src/main/java/com/palantir/conjure/java/lib/internal/ConjureDoubleList.java

```diff
@@ -55,6 +55,10 @@ public boolean addAll(int index, Collection<? extends Double> collection) {
         return delegate.addAllAtIndex(index, target);
     }
 
+    public void addAll(double... source) {
```

Comment:
nit: this isn't an override, so the public modifier doesn't buy us anything -- might as well be package private